docs/topics_5/web-reviewing-an-analysis.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+// * docs/web-console-guide_5/master.adoc
+[id='reviewing_an_analysis_{context}']
+= Reviewing an analysis
+
+In the *Review* screen, you can review your analysis and then decide to do one of the following: save your analysis without running it now, save  and run it now, or modify it before saving it.
+
```

Comment:
What on earth is this module about? It's not reviewing an analysis. It's deciding which button to click after creating an analysis. It looks like a step in a longer procedure.